src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+      </tr>
+      <tr>
+        <td class="numeric">17</td>
+        <td>opentype, svg, truetype, woff</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">18</td>
+        <td>opentype, truetype, woff, woff2</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">19</td>
+        <td>eot, truetype, woff, woff2</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+      <tr>
+        <td class="numeric">20</td>
+        <td>svg, woff</td>
+        <td class="numeric">0.0%</td>
+        <td class="numeric">0.0%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Top 20 des combinaisons de format de polices.") }}</figcaption>
+</figure>
+
+Cet ensemble de données semble indiquer que la majorité des gens utilisent déjà WOFF2 uniquement dans leurs blocs `@font-face`. Mais cela est bien sûr trompeur, si l'on se fie à notre discussion précédente sur la domination des polices Google dans l'ensemble des données. Google Fonts utilise des méthodes de détection pour servir un fichier CSS simplifié et n'inclut que le `format()` le plus moderne. Il n'est pas surprenant que WOFF2 domine les résultats ici pour cette raison, car la prise en charge de WOFF2 par les navigateurs est assez courante depuis un certain temps déjà.
+
+Il est important de noter que ces données particulières ne soutiennent ni ne contredisent vraiment l'idée d'utiliser uniquement WOFF2 pour le moment, mais cela reste une idée tentante.
+
+## Lutter contre le texte invisible
+
+Le premier outil dont nous disposons pour lutter contre le comportement par défaut de chargement des polices web soit l'"invisibilité pendant le chargement" (aussi appelé FOIT), est `font-display`. Ajouter `font-display : swap` à votre bloc `@font-face` est un moyen facile de dire au navigateur d'afficher le texte de remplacement pendant le chargement de la police web.
+
+La fonction <a hreflang="en" href="https://caniuse.com/#feat=mdn-css_at-rules_font-face_font-display">Support du navigateur</a> est également très utile. Internet Explorer et les versions antérieures à Chromium Edge ne sont pas pris en charge, mais ils affichent également un texte de remplacement par défaut lors du chargement d'une police Web (les FOIT ne sont pas autorisés ici). Pour nos tests dans Chrome, quelle est la fréquence d'utilisation de la fonction `font-display` ?
+
+{{ figure_markup(
+  caption="Pourcentage des pages mobiles qui utilisent le style <code>font-display</code>.",
+  content="26%",
+  classes="big-number"
+)
+}}
+
+Je suppose que cela va augmenter au fil du temps, surtout maintenant que <a hreflang="en" href="https://www.zachleat.com/web/google-fonts-display/">Google Fonts ajoute "font-display" à tous les nouveaux extraits de code</a> copiés de leur site.
```

Comment:
Peut-être mettre juste <code>font-display</code> au lieu de "font-display"?

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7